packages/e2e/local-network/scripts/mint-tokens.sh

```diff
@@ -47,7 +47,7 @@ done
 
 cardano-cli transaction build \
   --babbage-era \
-  --change-address "$faucetAddr" \
+  --change-address "$addr" \
```

Comment:
Bug? Next time it's best to commit separately, still as a `test` type, but it's part of a different change.

Reply:
The funds should stay in address for next script (mint-handle) to use them.

Reply:
Oh, so functional change to support this extension. Got it